File: Exercícios de Lógica/Python/Exercicio8.py
'''
Exercicio 008

Faça um Programa que pergunte quanto você ganha por hora e o número de horas trabalhadas no mês. Calcule e mostre o total do seu salário no referido mês.
'''

dinheiro = float(input("Quanto você ganha por hora: "))
horas = float(input("Quantas horas você trabalha por mês: "))

salario = dinheiro * horas

print(f"O seu salário é: R$ {salario} reais")

File: Exercícios de Lógica/Java/Extra/Exrcicios_Java/src/Exercicio8.java
/*
Exercicio 008
Faça um Programa que pergunte quanto você ganha por hora e o número de horas trabalhadas no mês.
Calcule e mostre o total do seu salário no referido mês.
 */

import java.util.Scanner;
public class Exercicio8 {
    public static void main(String[] args){
        Scanner scanner = new Scanner(System.in);

        System.out.print("Digte quanto você ganha por hora: ");
        float valorHora = scanner.nextFloat();

        System.out.print("Digite quantas horas você trabalha por mês: ");
        float horasTrabalhadas = scanner.nextFloat();

        float resultado = valorHora*horasTrabalhadas;
        System.out.println("Seu salário mensal é: R$ "+resultado+" reais");

        scanner.close();
    }

}
